docs/deployment/sso.md

```diff
@@ -66,85 +67,14 @@ SAML2.0 is configured through **Semgrep AppSec Platform**. To set up SSO:
 5. Select the box next to **This SSO supports non-password authentication mechanisms (e.g. MFA, X509, PasswordLessPhoneSignin)** if applicable.
 6. Click **Save** to proceed.
 
-If you are using Google Workspace SAML, see [SAML Single Sign-on with Google Workspace](/docs/kb/semgrep-appsec-platform/saml-google-workspace) for more specific guidance.
```

Comment:
I'd like it if we could retain a more obvious link for this page too. I thought this placement was most helpful, but above the general instructions would also work. Because there are other sections between the general SAML instructions and the references where the final link is, I think this gets a bit lost otherwise.

Reply:
I made several changes: I put this note back, but I put it at the top the way we did for Entra, both for consistency's sake and so people don't go through the steps, then realize they shouldn't. I also changed them to notes boxes so they aren't so in-your-face. Additionally, we added the admonition regarding admin accounts at the end here, and I thought two notes would be a bit much. WDYT?

Reply:
Yeah, this looks good!